changelog.md

```diff
@@ -1,5 +1,7 @@
 ## 3.1.0 [unreleased]
 
+* Fix default_parts config in Pages. [#3069](https://github.com/refinery/refinerycms/pull/3069). [Brice Sanchez](https://github.com/bricesanchez)
```

Comment:
This could be more descriptive, for example:

* Updated `Refinery::Pages.default_parts=` config to require both a `title` and a `slug` attribute for each part. [#3069](https://github.com/refinery/refinerycms/pull/3069). [Brice Sanchez](https://github.com/bricesanchez)